README.md

```diff
@@ -92,7 +92,21 @@ if (!global.Intl) {
 
 _note: the locale data is required for the polyfill to function when using it in a browser environment, in the example above, the english (`en`) locale is being required along with the polyfill itself._
 
+If you're using [webpack 2], instead of `require.ensure(...)` use `System.import`:
+
+```javascript
+// ...
+
+    Promise.all([
+        System.import('intl'),
```

Comment:
AFAIK webpack 2 is experimental. Also, I'm the editor of the Loader Spec, and `System.import` does not exist in that spec yet, only `System.loader.import`, not sure where is that coming from.